src/components/grid-aware/HomePageLargeParagraph/HomePageLargeParagraph.module.css

```diff
@@ -0,0 +1,59 @@
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: 1fr minmax(var(--grid-content-min-width), var(--grid-content-max-width)) 1fr;
+  justify-items: center;
+  margin: auto;
+  max-width: var(--grid-bleed-max-width);
+  padding-bottom: 120px;
+  padding-top: 120px;
```

Comment:
In the mobile design, these should both be 80px. Could you override these in the media query for this selector to match the design?

Reply:
👍 Added padding-bottom and padding-top with 80px to bleedWrapper media query.